code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -345,6 +345,12 @@ This function restores all limbs.
 
 	//visible_message("Hit debug. [damage]|[damagetype]|[def_zone]|[blocked]|[sharp]|[used_weapon]")
 
+	if(blocked >= 1) //Complete negation
```

Comment:
```suggestion
	if(blocked || !damage) //Complete negation
```

Reply:
No, blocked has to be greater than 1, unless decimal amounts don't count as being true.